lib/action-group.ts

```diff
@@ -53,6 +54,9 @@ export function isChildPath(parent: string, child: string): boolean {
 }
 
 function copyMatchesFile(copy: LocalCopy, file: string): boolean {
+	console.log('--------------------------------------------------');
+	console.log(copy.source);
+	console.log(file);
```

Comment:
leftover debug

Reply:
Ugh I had removed that, then didn't save the file...